config/routes/public/v1.rb

```diff
@@ -44,6 +44,18 @@ def self.extended(router)
               resources :works, only: :index
             end
 
+            resources :media_contents do
```

Comment:
I'm not sure we need the nested routes if at this point we're only looking for and endpoint that will give us the the list of `media_contents` for a project.